app/src/app/[lng]/cities/[cityId]/HIAP/page.tsx

```diff
@@ -51,8 +54,81 @@ export default function HIAPPage(props: {
             population={population}
           />
 
-          <Box></Box>
-          <Box display="flex" mx="auto" mt="80px" w="full" maxW="1090px">
+          <Box
+            display="flex"
+            mx="auto"
+            py="56px"
+            w="full"
+            maxW="1090px"
+            flexDirection="column"
+            gap="24px"
+          >
+            {/* citycatalyst actions section */}
```

Comment:
This could be a separate component instead of a section with a comment in a larger component